data-science-onramp/feature-engineering/citibike.ipynb

```diff
@@ -0,0 +1,385 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gcloud services enable ml.googleapis.com"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Load the data\n",
+    "\n",
+    "## Import libraries\n",
+    "Running the following cell will import the libraries needed. \n",
+    "\n",
+    "+ `Pandas`: to store and manipulate the dataset.\n",
+    "* `Google Cloud Storage`: to retrieve the dataset from the GCS bucket where the dataset is stored."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from google.cloud import storage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define constants\n",
+    "Define the name of your Google Cloud Storage bucket where the cleaned data is stored. \n",
+    "\n",
+    "+ `BLOB_PREFIX`: indicates the folder where the files are stored.\n",
+    "+ `DIR_NAME`: holds the name of the local folder where the files will be downloaded to."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BUCKET_NAME = 'your-bucket-name'\n",
+    "BLOB_PREFIX = 'clean_citibike_data.csv.gz/part'\n",
+    "DIR_NAME = 'citibike_data'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load the data\n",
+    "\n",
+    "Run the following command to create a local folder where the dataset files will be stored."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!mkdir $DIR_NAME"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following cell will retrieve all of the files with the `BLOB_PREFIX` defined above and download them into the folder. You will also create a list of the file names so they can be referenced later when loading the data into a dataframe."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create storage client\n",
+    "storage_client = storage.Client()\n",
+    "\n",
+    "# List files in the bucket with the specified prefix\n",
+    "blobs = storage_client.list_blobs(BUCKET_NAME, prefix=BLOB_PREFIX)\n",
+    "\n",
+    "# Go through the files and save them into the local folder\n",
+    "filenames = []\n",
+    "for i, blob in enumerate(blobs):\n",
+    "    filename = f'{DIR_NAME}/citibike{i}.csv.gz'\n",
+    "    blob.download_to_filename(filename)\n",
+    "    filenames.append(filename)\n",
+    "    print('Downloaded file: ' + str(blob.name))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, you can load the files into a dataframe. \n",
+    "\n",
+    "First, define the schema. From this dataset, you will need 4 columns:\n",
```

Comment:
nit - can you emphasize the names of the columns? Either by making them bold or using the code style 

Consider also having the bullet points be identical to the column names defined in the cell below

Reply:
Also, are these descriptions of the columns themselves or what we're going to do with them?

Reply:
I included a description of why we needed the columns, do you think this should be made more clear?